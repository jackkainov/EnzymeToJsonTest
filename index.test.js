import * as React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Test from './index';


describe('test', () => {
  it('should render snapshot', () => {
    const tree = shallowToJson(shallow(<Test />));
    
    expect(tree).toMatchSnapshot();
  });
});
