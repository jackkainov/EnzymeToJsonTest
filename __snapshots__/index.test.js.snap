exports[`test should render snapshot 1`] = `<InnerComponent />`;
